about/reports/2020_institution_submissions.md

```diff
@@ -0,0 +1,20 @@
+# Submissions by Institution in 2020
+
+_See more [arXiv in Numbers](/about/reports/2020_usage)_
+
+The following table shows submissions by institutions in 2018, 2019, and 2020. To search for a specific institutional domain, hover your mouse over the right hand corner of the list of institutions to reveal the search icon.
```

Comment:
@afromme we should indicate somewhere on this page that the data comes from Microsoft Analytics

Reply:
Yes, agreed and done